static/templates/items/weapon-details.html

```diff
@@ -89,12 +89,10 @@ <h3>
                 <option value=""></option>
                 {{#each preciousMaterials as |material materialKey|}}
                     <optgroup label="{{material.label}}">
-                        {{#each material as |grade gradeKey|}}
-                            {{#if grade.label}}
-                                <option value="{{materialKey}}-{{gradeKey}}"{{#if grade.selected}} selected="selected"{{/if}}>
-                                    {{localize "PF2E.Item.Weapon.MaterialAndRunes.MaterialOption" type=material.label grade=grade.label}}
-                                </option>
-                            {{/if}}
+                        {{#each material.grades as |grade gradeKey|}}
+                            <option value="{{materialKey}}-{{gradeKey}}"{{#if grade.selected}} selected="selected"{{/if}}>
```

Comment:
```suggestion
                            <option value="{{materialKey}}-{{gradeKey}}" {{selected grade.selected}}>
```

Reply:
I forgot to modernize it, but the correct thing is {{#select and a block. This will a refactor on the MaterialSheetData so that the value is exposed at the top level. I can do that though it'll take a bit.